examples/samples/sendersReceiversWithTopics.ts

```diff
@@ -0,0 +1,105 @@
+import {
+  OnMessage,
+  OnError,
+  MessagingError,
+  delay,
+  ServiceBusMessage,
+  generateUuid,
+  Namespace,
+  SendableMessageInfo,
+  SubscriptionClient,
+  TopicClient
+} from "../../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const topic = process.env.TOPIC_NAME || "";
+const subscription1 = process.env.SUBSCRIPTION_NAME1 || "";
+const subscription2 = process.env.SUBSCRIPTION_NAME2 || "";
+const subscription3 = process.env.SUBSCRIPTION_NAME3 || "";
+
+console.log(`str: ${str}`);
+console.log(`path: ${topic}`);
+console.log(`Subscription 1: ${subscription1}`);
+console.log(`Subscription 2: ${subscription2}`);
+console.log(`Subscription 3: ${subscription3}\n`);
+console;
+
+let ns: Namespace;
+
+async function sendMessages(topicClient: TopicClient): Promise<void> {
+  const data: { lastName: string; firstName: string }[] = [
+    { lastName: "Einstein", firstName: "Albert" },
+    { lastName: "Heisenberg", firstName: "Werner" },
+    { lastName: "Curie", firstName: "Marie" },
+    { lastName: "Hawking", firstName: "Steven" },
+    { lastName: "Newton", firstName: "Isaac" },
+    { lastName: "Bohr", firstName: "Niels" },
+    { lastName: "Faraday", firstName: "Michael" },
+    { lastName: "Galilei", firstName: "Galileo" },
+    { lastName: "Kepler", firstName: "Johannes" },
+    { lastName: "Kopernikus", firstName: "Nikolaus" }
+  ];
+
+  for (let index = 0; index < data.length; index++) {
+    const element = data[index];
+    const message: SendableMessageInfo = {
+      body: `${element.firstName} ${element.lastName}`,
+      label: "Scientist",
+      timeToLive: 2 * 60 * 1000, // 2 minutes
+      messageId: generateUuid()
+    };
+
+    console.log(`\n Sending ${message.body} \n`);
+    await topicClient.send(message);
+  }
+}
+
+async function main(): Promise<void> {
+  ns = Namespace.createFromConnectionString(str);
+  const client = ns.createTopicClient(topic);
+
+  const subscription1Client = ns.createSubscriptionClient(topic, subscription1);
+  const subscription2Client = ns.createSubscriptionClient(topic, subscription2);
+  const subscription3Client = ns.createSubscriptionClient(topic, subscription3);
+
+  async function setupReceiveHandlers(client: SubscriptionClient): Promise<void> {
+    const onMessage: OnMessage = async (brokeredMessage: ServiceBusMessage) => {
+      console.log(`subscription: ${client.name}  Retrieved: ${brokeredMessage.body}`);
+    };
+    const onError: OnError = (err: MessagingError | Error) => {
+      console.log("\n>>>>> Error occurred: ", err);
+    };
+
+    const rcvHandler = client.receive(onMessage, onError);
+    await delay(10000);
+    await rcvHandler.stop();
+  }
+
+  //Setting up receive handlers
+  setupReceiveHandlers(subscription1Client);
```

Comment:
Yes, I was using the similar pattern in topicsGettingStarted sample because I wanted to print the output like this:
![untitled](https://user-images.githubusercontent.com/45216704/50460716-49550e00-092d-11e9-8441-6c5f5c41ea9d.png)
That's why I'm setting up the receivers before sending and if we add await in `setupReceiveHandlers` methods, we won't be able to receive anything because we are calling `setupReceiveHandlers` before sending.

@HarshaNalluru I would suggest to send before receive and add await.

Reply:
@AlexGhiondea @HarshaNalluru I was following the .Net sample but I just changed the code for topicsGettingStarted sample in this PR https://github.com/Azure/azure-service-bus-node/pull/124 and followed the same pattern as others.

@HarshaNalluru I would suggest to use updated topicGettingStarted sample for the referral.

Reply:
Thanks @ShivangiReja!